src/storage/secondary/column.rs

```diff
@@ -195,18 +196,17 @@ impl Column {
                 "block is smaller than header size",
             ));
         }
-        let mut header = &block[..BLOCK_HEADER_SIZE];
-        let block_data = &block[BLOCK_HEADER_SIZE..];
+        let mut header = &block[block.len() - BLOCK_HEADER_SIZE..];
```

Comment:
should BLOCK_HEADER_SIZE be changed to BLOCK_FOOTER_SIZE?

Reply:
Shall the struct name `BlockHeader` be renamed as `BlockFooter` as well?

Reply:
Yes, +1 for this!

Reply:
Or BlockMeta, whatever you want.